deck.gl/examples/pure-js/vue/carto3/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "deckgl-for-vue",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "serve": "vue-cli-service serve",
+    "build": "vue-cli-service build",
+    "lint": "vue-cli-service lint"
+  },
+  "dependencies": {
+    "@deck.gl/carto": "8.5.0",
```

Comment:
^8.6.0

Reply:
Updated to 8.6.5. Sometimes there are breaking changes in minor versions, so I prefer to pin the example to a specific version.